### PR TITLE
Admin update room

### PR DESCRIPTION
**Pull request by ducbet**

### CODE REVIEW

File: app/controllers/admin/rooms_controller.rb [hunk 1]
```diff
@@ -31,6 +35,21 @@ def show
...
 
+  def edit
+    @room = Room.find_by id: params[:id]
+    @support = editable_room @room
+  end
+
+  def update
+    @room = Room.find_by id: params[:id]
+    if @room.set_map params_map, params_deleted_seats
+      redirect_to admin_room_url(@room)
```
Comment (thaotonto) [inline app/controllers/admin/rooms_controller.rb]: ()

File: app/controllers/admin/rooms_controller.rb [hunk 2]
```diff
@@ -31,6 +35,21 @@ def show
...
+    else
+      flash[:danger] = t "flash.edit_faild"
+      redirect_to edit_admin_room_url(@room)
```
Comment (thaotonto) [inline app/controllers/admin/rooms_controller.rb]: same

File: app/views/admin/rooms/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<!--
+Author: W3layouts
+Author URL: http://w3layouts.com
+License: Creative Commons Attribution 3.0 Unported
+License URL: http://creativecommons.org/licenses/by/3.0/
+-->
+<%= stylesheet_link_tag "room_template" %>
+<% provide :button_text, t(".button_text") %>
+<% provide :url, admin_room_path(id: @support.room.id) %>
```
Comment (thaotonto) [inline app/views/admin/rooms/edit.html.erb]: path(@support.room) thôi là đc. maybe

File: app/views/admin/rooms/show.html.erb [hunk 4]
```diff
@@ -25,7 +25,8 @@ License URL: http://creativecommons.org/licenses/by/3.0/
           <li>: <%= @room.seat_no %></li>
         </ul>
         <div class="clear"></div>
-          <%= link_to t(".change"),"#", class:"checkout-button" %>
+          <%= link_to t(".change"), edit_admin_room_path(@room.id),
```
Comment (thaotonto) [inline app/views/admin/rooms/show.html.erb]: @room

**Comment by ducbet** [comment]
image

**Comment by thaotonto** [comment]
ok

**Comment by noobsi** [comment]
ok

**Pull request closed without merging**